samples/azure-quantum/simulators/SparseSimulator.ipynb

```diff
@@ -0,0 +1,2274 @@
+{
```

Comment:
This sample looks to be in the wrong folder? `/samples/azure-quantum` is intended for service-oriented samples, but this is all about local usage.

Reply:
What would be the right folder for the `SparseSimulator.ipynb`?

Reply:
I'm not quite sure, let me think. In general, we haven't had samples decidated to individual simulators, so much as how to use those simulators in more user-focused samples. I'd honestly prefer that approach here as well, but I understand that design constraints preclude that. Perhaps the right approach is to rename the sample to something like "simulating quantum programs with large registers," and place in `getting-started/`?